services/src/main/scala/cromwell/pipeline/service/RunService.scala

```diff
@@ -37,7 +36,7 @@ object RunService {
           userId = userId,
           results = runCreateRequest.results
         )
-        runRepository.addRun(newRun)
+        projectService.getUserProjectById(runCreateRequest.projectId, userId).flatMap(_ => runRepository.addRun(newRun))
```

Comment:
If I'm right `addRun` will now cause two more exceptions from `getUserProjectById`. Shouldn't exceptions be handled more informatively  in `RunController`'s `addRun` from now on?

Reply:
Yep. we can add more exceptions, but I think we should do this on new task.